.changeset/pink-onions-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': minor
+---
+
+**BREAKING** Completely removed the `EntitySystemDiagramCard` component which was previously deprecated. `EntityCatalogGraphCard` should be used instead, as references to `EntitySystemDiagramCard` will now break.
```

Comment:
Let's call out that you import the `EntityCatalogGraphCard` from the `@backstage/plugin-catalog-graph` package as well, just in case

Reply:
Done, thanks.